lib/redis/alfred.rb

```diff
@@ -7,8 +7,8 @@ class << self
     # key operations
 
     # set a value in redis
-    def set(key, value)
-      $alfred.with { |conn| conn.set(key, value) }
+    def set(key, value, nx: false, ex: false) # rubocop:disable Naming/MethodParameterName
+      $alfred.with { |conn| conn.set(key, value, nx: nx, ex: ex) }
     end
 
```

Comment:
lets add a comment over setex  with a  TODO: deprecation and recommending usage of `set`